src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -6,9 +6,31 @@
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LENGTH_DATA = 6;
 
     @Override
     public User register(User user) {
-        return null;
+        if (checkLoginIfPresent(user) != null ) {
+            throw new InvalidDataException("User with this login has been already registered");
+        } else if (user.getAge() == null) {
```

Comment:
The 'age' field of the User class is of primitive type 'int', so it cannot be null. This condition will always be false and is unnecessary. Instead, you should check if the age is less than the minimum age.